tutor/commands/jobs_utils.py

```diff
@@ -0,0 +1,133 @@
+"""
+This module provides utility methods for tutor `do` commands
+
+Methods:
+- `get_mysql_change_charset_query`: Generates MySQL queries to upgrade the charset and collation of tables
+"""
+
+
+def get_mysql_change_charset_query(
+    database: str,
+    charset: str,
+    collation: str,
+    query_to_append: str,
+    charset_to_upgrade_from: str,
+) -> str:
+    """
+    Helper function to generate the mysql query to upgrade the charset and collation of tables
```

Comment:
it is not just tables, it is columns, tables and database.

Reply:
Updated.